src/main/perl/Filesystem.pm

```diff
@@ -155,6 +156,83 @@ sub remove_if_needed
     return $?;
 }
 
+
+=pod
+
+=head2 get_disk_uuid
+
+Fetches the (PART)UUID of the device with blkid 
+
+=cut
+
+sub get_disk_uuid
```

Comment:
this method belongs in teh module that handles the holding device (in this case Partition and or Disk)